suricata.yaml.in

```diff
@@ -160,9 +160,23 @@ outputs:
             # IP/UDP/TCP length values, and other events that render the packet invalid for further processing 
             # or describe unexpected behavior on an established stream. Networks which experience high
             # occurrences of anomalies may experience packet processing degradation.
-
-            # Enable dumping of packet header
-            # packethdr: no            # enable dumping of packet header
+            #
+            # Anomalies are reported for the following:
+            # 1. Packet: Values and conditions that are detected while decoding individual packets. This includes
+            # invalid or unexpected values for low-level protocol lengths as well as stream related events (TCP 3-way
+            # handshake issues, unexpected sequence number, etc).
+            # 2. Application level: These denote application layer specific conditions that are unexpected, invalid or
+            # are unexpected given the application monitoring state.
```

Comment:
Can we distinguish between decoder and stream events too?

Reply:
Of course. Let me know if you want that included with this work.

Reply:
What works best for you. I would like to get the stream events logic in soon so it can go in to 5.0rc1

Reply:
Yeah, I think it is likely that ppl will want to disable stream events separately from the decoder events.

Reply:
> Should this be a one-to-one mapping with the anomaly.type field in the logged output? So we'd have:
> 
> * packet
> * applayer
>   etc...
> 
> This is how eve logging is done where event_type matches the types list.

I'll ensure that the "type" matches one of decode, stream, application-layer in the reworked pr.